### PR TITLE
Update 'Whats new' and roadmap for v4.4.0 release

### PR DESCRIPTION
**Pull request by 36degrees**

### CODE REVIEW

File: views/partials/_whats-new.njk [hunk 1]
```diff
@@ -6,9 +6,10 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-<p class="govuk-body">17 October 2022: We’ve changed our ‘Backlog’ page into the new ‘<a href="/community/upcoming-components-patterns/" class="govuk-link">Upcoming components and patterns</a>’ page and chosen 3 priorities that we plan to work on next.
-</p>
-        <p class="govuk-body"><a href="https://mailchi.mp/d484adee17c1/email-updates" class="govuk-link">Sign up to get update emails about the Design  System</a>.</p>
```
Comment (domoscargin) [inline views/partials/_whats-new.njk:11]: Was this line meant to be deleted?

Comment (36degrees) [reply views/partials/_whats-new.njk:11]: Nope! 🤦🏻

Comment (36degrees) [reply views/partials/_whats-new.njk:11]: I've put it back now 👀

**Review by domoscargin — APPROVED** [review]
Looks good to me!

#ThisIsGettingClose

**Pull request merged**